js/controllers/list/submissions/SubmissionsListItem.vue

```diff
@@ -37,10 +41,10 @@
 		<div v-else class="pkpListPanelItem--submission__stage">
 			<div class="pkpListPanelItem--submission__stageRow">
 				<div class="pkpListPanelItem--submission__stageLabel">
-					<template v-if="submission.status.id === 3 || submission.status.id === 4">
-						{{ submission.status.label }}
+					<template v-if="item.status.id === 3 || item.status.id === 4">
```

Comment:
Sure. We do already have a mechanism for getting constants over to JS-land -- see the `define_exposed` PHP function -- though I'm open to ideas for improving/replacing it.